<?xml version="1.0"?>
<launch>
	<!-- ### add parameters ### -->
	<!-- <arg name="server_name" default="$(env HOME)"/> -->
	<!-- <arg name="deploy_dir" default="$(arg server_name)/Code/thesis_ws/src/path_planner"/> -->
	<arg name="deploy_dir" default="/home/jialiang/Code/thesis_ws/src/hybrid_astar"/>
	<node name="node_path_evaluator" pkg="hybrid_astar" type="node_path_evaluator" output="screen" args="--log_dir $(arg deploy_dir)/log --minloglevel 0 --max_log_size 100 --alsologtostderr" launch-prefix="xterm -e gdb -ex run --args "></node>


</launch>